src/app/routes/broker-ui/catalog-page/catalog-page/catalog-page.component.html

```diff
@@ -43,21 +43,10 @@
       [pageIndex]="state.pageZeroBased"
       [length]="state.paginationMetadata.numTotal"
       (page)="onPageChange($event)"></mat-paginator>
-
-    <!-- Sorting Select-->
-    <mat-form-field
-      *ngIf="state.sortings.length"
-      class="mat-form-field-without-margin-bottom"
-      color="accent">
-      <mat-label>Sort By</mat-label>
-      <mat-select
-        [compareWith]="'sorting' | compareByField"
-        [formControl]="sortBy">
-        <mat-option *ngFor="let item of state.sortings" [value]="item">
-          {{ item.title }}
-        </mat-option>
-      </mat-select>
-    </mat-form-field>
+    <!-- View Switch -->
+    <view-selection
+      [selected]="viewMode"
+      (onSelectionChange)="onViewSelectionChange($event)"></view-selection>
```

Comment:
could be shortened to `[(selected)]="viewMode"`, if the output's called "selectedChange"